projects/igniteui-angular/src/lib/directives/autocomplete/autocomplete.directive.ts

```diff
@@ -277,7 +277,7 @@ export class IgxAutocompleteDirective extends IgxDropDownItemNavigationDirective
         if (this.disabled || !this.collapsed) {
             return;
         }
-        this.target.width = this.parentElement.clientWidth + 'px';
+        this.target.width = this.target.width || (this.parentElement.clientWidth + 'px');
```

Comment:
Please add a comment describing that if not defined width of the drop down will be the width of the input the autocomplete is attached to. You can add also a test.